parts/k8s/addons/1.18/kubernetesmasteraddons-kube-proxy-daemonset.yaml

```diff
@@ -9,6 +9,15 @@ data:
       kubeconfig: /var/lib/kubelet/kubeconfig
     clusterCIDR: "{{ContainerConfig "cluster-cidr"}}"
     mode: "{{ContainerConfig "proxy-mode"}}"
+    {{- if ContainerConfig "bind-address"}}
```

Comment:
So go template if operates against strings according to `""` = false and any non-empty string = true ?

Reply:
@jackfrancis correct, if it's `""` then it's ignored.

Reply:
Looked up and verified in the actual docs as well :)